### ISSUE TITLE
Cannot  create spring  project out of it

### ISSUE THREAD

**Issue by Muhib007dev** [jdegre/5GC_APIs#36]
I am having trouble converting these YAML files into spring project. Every time I face different issues like some models isn't converted or even if it does when I run the project some attributes are missing from the API. I tried open API tools, swagger, etc. Nothing seems to work out. Can you tell me which version or which tool should I use to convert it to a spring project?

**Comment by MysterionRise** [comment]
@Muhib007dev  I would recommend to use https://openapi-generator.tech/docs/generators#server-generators and select spring as target. That worked for me just fine

**Comment by Muhib007dev** [comment]
Yup, I did the same thing and had some issues in that as well, so I just worked around with it. Thank you for the suggestion.
